setup/install/deploy.md

```diff
@@ -39,20 +39,27 @@ deploy apply`.
 
 ## Connect to the Spinnaker UI
 
-If you have not enabled any sort of authentication, Spinnaker will not be
-publicly reachable by default. In this case, you will need to run the
-following command to reach the UI on [localhost:9000](http://localhost:9000):
+1. Run the following command:
 
-```bash
-hal deploy connect
-```
+   ```bash
+   hal deploy connect
+   ```
+
+   If necessary, set up an SSH tunnel to the host running Halyard.
+
+   This  command automatically forwards ports 9000 (Deck UI) and 8084 (Gate API
+     service).
+
+1. Navigate to [localhost:9000](localhost:9000).
+
+### Alternatives
 
-If you want to make Spinnaker publicly reachable without running that command,
-please read the [Halyard FAQ](/setup/quickstart/faq/).
+* You can make Spinnaker publicly reachable without running this command,
+as described
+[here](/setup/quickstart/faq//#i-want-to-expose-localdebian-spinnaker-on-a-public-ip-address-but-it-always-binds-to-localhost)
```

Comment:
Worth mentioning in the link text that one is for localdebain, and the other is for k8s?

Reply:
Aye. Done.